content/en/blog/releases/changes-11-april-2018.md

```diff
@@ -15,13 +15,13 @@ We announced [Jenkins X](https://jenkins.io/blog/2018/03/19/introducing-jenkins-
  
  We've made a lot of progress and improvements! The rest of the blog goes into more detail. 
  
- But first I'd like to call out a few specific details from the [Jenkins X Futures](/contribute/roadmap/):
+ But first I'd like to call out a few specific details from the [Jenkins X Futures](https://jenkins-x.io/contribute/roadmap/):
```

Comment:
this links to an old page that i hope to someday get rid of :) the content itself is outdated and it's one of the pages from the old design that somehow hasn't been remove from the production website.

since the page no longer exists, it might be best to simply remove the link?  there shouldn't be that many of these links